SonarQube.Common/EncodingProviderWrapper.cs

```diff
@@ -0,0 +1,25 @@
+﻿//-----------------------------------------------------------------------
```

Comment:
Not sure if you are using SonarLint for VS but you should have a warning saying the header is incorrect.